### ISSUE TITLE
For anything other than WordPress transfers, redirect them to the design picker

### ISSUE THREAD

**Issue by vishnugopal** [Automattic/wp-calypso#65335]

**Comment by vishnugopal** [comment]
Discussion:
- Can we redirect to the design picker without setting the vertical?

**Comment by jimmy319** [comment]
### Spike: redirect to the design picker directly without setting the vertical

<img width="1047" alt="Screen Shot 2022-07-21 at 2 59 23 PM" src="https://user-images.githubusercontent.com/1024985/180152365-0a311746-ae38-4970-b16d-3dd42896a06f.png">
<img width="1782" alt="Screen Shot 2022-07-21 at 3 03 53 PM" src="https://user-images.githubusercontent.com/1024985/180152431-a4cd1fc0-abd7-4635-9502-6477c79d970a.png">

We can't redirect to the design picker (the one with generated designs) directly without [setting the siteVerticalId via the verticals screen](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/index.tsx#L45-L48). The `Themes` screen will appear if the [siteVerticalId is not being set](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/site-setup-design-picker.tsx#L115-L119).

I also tried to call the design preview generation API without presenting the `vertical_id` parameter and got a [mock preview page](https://public-api.wordpress.com/wpcom/v2/block-previews/site?stylesheet=pub%2Fheiwa&pattern_ids=2077%2C2080%2C2082%2C2144%2C2088&header_pattern_ids=&footer_pattern_ids=&vertical_id=&language=en&viewport_width=1600&viewport_height=1040&source_site=patternboilerplates.wordpress.com&use_screenshot_overrides=true&site_title=Education). ([A normal design preview should look like this](https://public-api.wordpress.com/wpcom/v2/block-previews/site?stylesheet=pub%2Fheiwa&pattern_ids=2077%2C2080%2C2082%2C2144%2C2088&header_pattern_ids=&footer_pattern_ids=&vertical_id=74&language=en&viewport_width=1600&viewport_height=1040&source_site=patternboilerplates.wordpress.com&use_screenshot_overrides=true&site_title=Education))

### To sum up

We can redirect users to the design picker directly if displaying the Themes screen meets our requirements. If not, we have to redirect users to the vertical selection screen first and then proceed to the design picker.

**Comment by vishnugopal** [comment]
As discussed, let's spike this out by redirecting to the themes screen, but also keep a watch on the [unified design picker project](pbxlJb-25e-p2).
